spec/spec_helpers/fixture_helper.rb

```diff
@@ -0,0 +1,38 @@
+module FixtureHelper
+  def in_dest_dir(*files)
+    dest_dir.join(*files)
+  end
+
+  def dest_dir
+    @dest_dir ||= Pathname.new(File.expand_path("../tmp/dest", File.dirname(__FILE__)))
+  end
+
+  def source_dir
+    @soruce_dir ||= Pathname.new(File.expand_path("../test-site", File.dirname(__FILE__)))
+  end
+
+  def config_defaults
+    {
+      "source"      => source_dir.to_s,
+      "destination" => dest_dir.to_s,
+      "plugins"     => ["jekyll-github-metadata"],
+      "gems"        => ["jekyll-github-metadata"],
+    }
+  end
+
+  def make_page(options = {})
```

Comment:
It's used both in the fixture helper and in the edit link tag spec to create the context, which needs the page object, not the page data.